examples/circular_infra/succession.json

```diff
@@ -0,0 +1,3 @@
+{
+    "successions": []
```

Comment:
It should be great to make `succession.json` file optional to avoid having empty files like that